### PR TITLE
Populate resource aliases, update README

### PR DESCRIPTION
**Pull request by jpeimer**
- Populate resource_aliases
- Add script to update the resource_aliases
- Update README

## Release Notes

- **New Features**
  - Introduced the `must-gather-explorer` CLI tool for exploring collected must-gather data.
  - Added the `update-resources-aliases` CLI command for managing Kubernetes resource aliases.

- **Documentation**
  - Overhauled the `README.md` to provide detailed usage instructions and installation guidance for the Must-gather explorer tool.

- **Data Structure**
  - Added `aliases.json` for mapping Kubernetes resource types to their aliases and shorthand notations.

- **Dependencies**
  - Updated `pyproject.toml` to include `pyhelper-utils` and added a script for resource alias management.

### CODE REVIEW

**Comment by myakove** [comment]
pre-commit.ci run

File: app/resource_aliases.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    aliases_file_path = "app/manifests/aliases.json"
+
+    try:
+        with open(aliases_file_path) as aliases_file:
+            resources_aliases = json.load(aliases_file)
+    except Exception:
+        resources_aliases = {}
+
+    cli_resource_alias: Dict[str, List[str]] = {}
```
[the remark this answers is not in the document]
Comment (jpeimer) [reply app/resource_aliases.py]: 1 - Done, 2 - Already have a in the next PR

File: app/resource_aliases.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        # Fill cli_resource_alias dict with {Kind}:['alias1','alias2','...']
+        cli_resource_alias[split_line[-1]] = alias_list
+        # cli_resource_alias = {'VirtualMachine': ['virtualmachines','vm','vms']}
+
+    resources_aliases.update(cli_resource_alias)
+
+    # Write to file
+    with open(aliases_file_path, "w") as aliases_file:
+        json.dump(resources_aliases, aliases_file, indent=4)
```
[the remark this answers is not in the document]
Comment (jpeimer) [reply app/resource_aliases.py]: Done

File: app/manifests/aliases.json [hunk 3]
```diff
@@ -0,0 +1,1059 @@
...
+    "User": [
+        "users"
+    ],
+    "IPPool": [
+        "ippools"
+    ],
+    "OverlappingRangeIPReservation": [
+        "overlappingrangeipreservations"
+    ]
+}
```
[the remark this answers is not in the document]
Comment (jpeimer) [reply app/manifests/aliases.json:1059]: In the next PR, I'm converting all to a lower case, will address it there

[the remark this answers is not in the document]
Comment (jpeimer) [reply app/manifests/aliases.json:1059]: No, thanks

**Review by myakove — APPROVED** [review]

**Pull request merged**